Jenkinsfile

```
@@ -199,48 +190,53 @@ pipeline {
                         stage("unit-test/${it.join('-')}") {
                           try {
                             printVariation(it)
-                            if (ARCH=="HSW" && PARTYPE=="PAR") {
-                              sh "mpirun -n 4 ./src/${it.join('-')} -t -d ./test_input/"
-                            } else if (ARCH=="HSW" && PARTYPE=="SEQ") {
-                              sh "./src/${it.join('-')} -t -d ./test_input/"
-                            } else if (ARCH=="KNL" && PARTYPE=="PAR") {
-                              sh """
-                                source /etc/profile.d/modules.sh
-                                export OMP_NUM_THREADS=10
-                                export I_MPI_PMI_LIBRARY=/usr/lib64/libpmi.so
-                                while : ; do
-                                  set +e
-                                  output=`srun -n 2 --time=00:05:00 ./src/${it.join('-')} -t -d ./test_input/ 2>&1`
-                                  rc=\$?
-                                  if [[ \$rc == 1 && (\$output == *"Job violates accounting/QOS policy"* || \$output == *"Socket timed out on send/recv"*)]] ; then
-                                    echo "srun submit limit reached or socket timed out error, trying again in 60s"
-                                    sleep 60
-                                    continue
-                                  fi
-                                  set -e
-                                  echo \$output
-                                  exit \$rc
-                                done
-                              """
-                            } else if (ARCH=="KNL" && PARTYPE=="SEQ") {
-                              sh """
-                                source /etc/profile.d/modules.sh
-                                export OMP_NUM_THREADS=10
-                                export I_MPI_PMI_LIBRARY=/usr/lib64/libpmi.so
-                                while : ; do
-                                  set +e
-                                  output=`srun -n 1 --time=00:05:00 ./src/${it.join('-')} -t -d ./test_input/ 2>&1`
-                                  rc=\$?
-                                  if [[ \$rc == 1 && (\$output == *"Job violates accounting/QOS policy"* || \$output == *"Socket timed out on send/recv"*)]] ; then
-                                    echo "srun submit limit reached or socket timed out error, trying again in 60s"
-                                    sleep 60
-                                    continue
-                                  fi
-                                  set -e
-                                  echo \$output
-                                  exit \$rc
-                                done
-                              """
+                            def legacyCellProcessorOptions = (VECTORIZE_CODE == "NOVEC" ? [false, true] : [false])
```

```suggestion
                            def legacyCellProcessorOptions = (VECTORIZE_CODE == "NOVEC" and REDUCED_MEMORY_MODE == "0" ? [false, true] : [false])
```
should also be false if REDUCED_MEMORY_MODE is activated

---

Right, thanks